scripts/config-secrets.sh

```
@@ -0,0 +1,86 @@
+
+  if [[ -f "$ARR_ENV_FILE" ]] && [[ "$FORCE_ROTATE_API_KEY" != "1" ]]; then
+    local existing
+    existing="$(get_env_kv "GLUETUN_API_KEY" "$ARR_ENV_FILE" || true)"
+    if [[ -n "$existing" ]]; then
+      GLUETUN_API_KEY="$existing"
+      msg "Using existing API key"
+      return
+    fi
+  fi
+
+  GLUETUN_API_KEY="$(safe_random_alnum 64)"
```

Shellcheck (0.11.0)

[warning] GLUETUN_API_KEY appears unused. Verify use (or export if used externally).

(SC2034)